actions.go

```diff
@@ -178,3 +186,21 @@ func showPaged(s *Shell, text string) error {
 	cmd.Stdin = bytes.NewBufferString(text)
 	return cmd.Run()
 }
+
+func showPagedReader(s *Shell, r io.Reader) error {
```

Comment:
I think it'll be better to reuse this function in `showPaged` rather than have almost same code in both functions.

Reply:
True ! I changed the code in the branch so ShowPaged use showPagedReader